src/status_im/chat/handlers/receive_message.cljs

```diff
@@ -28,13 +28,14 @@
               message-id timestamp clock-value]
        :as   message
        :or   {clock-value 0}}]
-  (let [same-message     (messages/get-by-id message-id)
-        current-identity (get-current-identity db)
-        chat-id'         (or group-id chat-id from)
-        exists?          (chats/exists? chat-id')
-        active?          (chats/is-active? chat-id')
-        local-clock      (messages/get-last-clock-value chat-id')
-        clock-new        (clocks/receive clock-value local-clock)]
+  (let [same-message      (messages/get-by-id message-id)
+        current-identity  (get-current-identity db)
+        chat-id'          (or group-id chat-id from)
+        exists?           (chats/exists? chat-id')
+        active?           (chats/is-active? chat-id')
+        local-clock       (messages/get-last-clock-value chat-id')
+        clock-new         (clocks/receive clock-value local-clock)
+        recipient-pub-key (-> message :content :params :bot-db :public :recipient :whisper-identity)]
```

Comment:
maybe better to use `get-in` here

Reply:
I typically use `get-in` when there's a good chance we'll have non keyword keys (looks better than `(get "key-name")` in threading).  Is there any specific reason you think of or aesthetic?

Reply:
`get-in` is more readable, so if i see `get-in` i don't need to think, i know exactly what it does, when i see thread macro i need time to understand what this macro does in that case

Reply:
Fair enough